spec/classes/katello_qpid_spec.rb

```diff
@@ -44,7 +55,7 @@
             .with_ssl_key('/etc/pki/katello/private/foo.example.com-qpid-broker.key')
         end
 
-        ['entitlement.created', 'entitlement.deleted', 'pool.created', 'pool.deleted', 'compliance.created'].each do |binding|
+         @events.each do |binding|
```

Comment:
You could maybe do `let` instead, `@events` doesn't seem to be defined here.

```ruby
let :events do
  [
    'entitlement.created',
    'entitlement.deleted',
    'pool.created',
    'pool.deleted',
    'compliance.created',
    'system_purpose_compliance.created'
  ]
end
```

Reply:
That's what I tried the first time but let me put the let in a different spot and try again